packages/hawtio/src/plugins/camel/contexts/Contexts.tsx

```diff
@@ -0,0 +1,138 @@
+import { Card, CardBody, Text } from '@patternfly/react-core'
+import { InfoCircleIcon } from '@patternfly/react-icons'
+import { OnRowClick, Table, TableBody, TableHeader, TableProps, wrappable } from '@patternfly/react-table'
+import { IResponse } from 'jolokia.js'
+import { AttributeValues } from '@hawtiosrc/plugins/connect/jolokia-service'
+import { useEffect, useState, useContext } from 'react'
+import { PluginNodeSelectionContext } from '@hawtiosrc/plugins'
+import { log } from '../globals'
+import { contextsService, ContextAttributes } from './contexts-service'
+import { moveElement } from '@hawtiosrc/util/arrays'
+import { isObject } from '@hawtiosrc/util/objects'
+
+export const Contexts: React.FunctionComponent = () => {
```

Comment:
Same here. We should revisit if we can rename it to singular "Context" and drop presumption that we may have multiple contexts if it makes sense.

Reply:
Issue posted to track: https://github.com/hawtio/hawtio-next/issues/126